tests/tests_suite_test.go

```diff
@@ -605,7 +606,7 @@ func retrieveNodeArchitecture() string {
 	nodes := nodeList.Items
 	Expect(nodes).NotTo(BeEmpty(), "No nodes found")
 	architecture := nodes[0].Status.NodeInfo.Architecture
-	Expect(architecture).To(BeElementOf([]string{s390x, amd64, arm}), "Unsupported architecture")
+	Expect(architecture).To(BeElementOf([]string{s390x, amd64, arm, aarch64}), "Unsupported architecture")
```

Comment:
should we remove all the arch's dependency and checks?

Reply:
For sure it is one of the possible approach. I would like to give others some space to comment. Maybe there was a reason, why this check was introduced